streamtools/df/streamspraydf.py

```diff
@@ -35,7 +35,11 @@ def __init__(self,progenitor_mass,progenitor=None,pot=None,
            meankvec= (Fardal+2015-ish defaults) 
            
            sigkvec= (Fardal+2015-ish defaults) 
+
+           pot = (None) potential for integrating orbits
            
+           rtpot = (None) potential for calculating tidal radius and vircular velocity
```

Comment:
vircular -> circular

I would also write
"rtpot = (pot) potential for calculating tidal radius and vircular velocity"
to make it clear that the default is pot